ekncontent/tests/ekncontent/testUtils.js

```diff
@@ -10,4 +10,10 @@ describe('Utils', function () {
             }).not.toThrow();
         });
     });
+
+    describe('get_ekn_version', function () {
```

Comment:
No, in the commit message. but the review comment would disappear if I put it anywhere else besides this line :-P